task/runner.go

```diff
@@ -28,13 +28,13 @@ const (
 
 // Humanized errors
 var (
-	errFileInaccessible = errors.New("file could not be imported from URL because it was not accessible")
+	errFileInaccessible = UnretriableError{errors.New("file could not be imported from URL because it was not accessible")}
```

Comment:
The other two make sense, but could this one potentially be temporary for e.g a flakey HTTP endpoint?

Reply:
<img src="https://media4.giphy.com/media/0Av9l0VIc01y1isrDw/giphy.gif"/>